src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+        content="width=device-width,
+         user-scalable=no,
+         initial-scale=1.0,
+         maximum-scale=1.0,
+         minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <main class="card" data-qa="card">
+      <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="imac"
+      >
+
+      <p class="card__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </p>
+
+      <p class="card__code">
+          Код товара: 195434
+      </p>
+
+      <div class="card__comments">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__comments-count">
+          Отзывов: 5
+        </p>
+      </div>
+
+      <div class="card__prise">
+        <p class="card__prise-name">
+          Цена:
+        </p>
+
+        <p class="card__prise-uah">
+          69 999 грн
+        </p>
+      </div>
+
+      <button class="card__button"
+              data-qa="hover"
+      >
+        Купить
+      </button>
```

Comment:
On task description said that it should be a link

```suggestion
      <button
        class="card__button"
        data-qa="hover"
      >
        Купить
      </button>
```